lang/de/lifecyclestep_notifystudents.php

```diff
@@ -0,0 +1,69 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Lang strings for notify students step
+ *
+ * @package lifecyclestep_notifystudents
+ * @copyright  2021 Aaron Koßler WWU
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+$string['pluginname'] = 'Studierende-benachrichtigen-Schritt';
+$string['responsetimeout'] = 'Zeit, die der Lehrende hat, um zu reagieren';
+$string['option'] = 'Option';
+$string['option_help'] = '<p>' . 'Sie können zwischen den folgenden Optionen wählen:'
+    . '<br>' . 'Opt-In: Studierende werden nicht benachrichtigt außer wenn sich der Lehrende dafür entscheidet.'
```

Comment:
Ich glaube hier und in Zeile 30 kommt theoretisch vor dem außer ein Komma. (Da das nur der Admin sieht, ist das nicht so wichtig, aber ja).

Warum hast du hier in der Datei größtenteils Umlaute vermieden, gab es da Darstellungsfehler?

Reply:
Irgendwie hatte ich noch im Kopf, dass beim es da beim Projektseminar Probleme gab, aber vielleicht irre ich mich da auch. Den Komma Fehler habe ich korrigiert.